lib/Linter.js

```diff
@@ -135,4 +148,36 @@ function toLowerCase(string) {
 
 function toSemverMinor(executionPlatformVersion) {
   return executionPlatformVersion.split('.').slice(0, 2).join('.');
+}
+
+async function createCache(configName) {
+  let config = require('bpmnlint-plugin-camunda-compat').configs[ configName ];
+
+  if (!config) {
+    config = {
+      rules: {}
+    };
+  }
+
+  const rules = await requireRules(config);
+
+  return {
+    [ `config:bpmnlint-plugin-camunda-compat/${ configName }` ]: config,
+    ...rules
+  };
+}
+
+async function requireRules({ rules }) {
```

Comment:
If it doesn't use `await`, it shouldn't be `async`.

Reply:
Leftover I forgot to remove.

Reply:
Done 👍🏻